src/app_service/service/transaction/service.nim

```diff
@@ -390,13 +391,13 @@ QtObject:
       mtCommand = MultiTransactionCommandDto(
         fromAddress: from_addr,
         toAddress: to_addr,
-        fromAsset: tokenSymbol,
-        toAsset: tokenSymbol,
+        fromAsset: token.symbol,
+        toAsset: token.symbol,
         multiTxType: transactions.MultiTransactionType.MultiTransactionSend,
       )
 
     if self.isCollectiblesTransfer(sendType):
-      let contract_tokenId = mtCommand.toAsset.split(":")
+      let contract_tokenId = assetKey.split(":")
```

Comment:
not related to this task, but we should probably return with an error if the following check fails